docs/usage/filter.md

```diff
@@ -0,0 +1,13 @@
+---
+id: filter
+title: Filter
+---
+
+
+## Grayscale
+
+The `mono` parameter can be used to filter the image to Grayscale by giving it the value `000000`.
+
+| `?w=500&h=250` | `?w=500&h=250&mono=000000`|
+|:---:|:---:|
+| ![image](https://kdarkroom.herokuapp.com/sample-image.jpg?w=500&h=250) | ![image](https://kdarkroom.herokuapp.com/sample-image.jpg?w=500&h=250&mono=000000) |
```

Comment:
Can we get this host value `kdarkroom.herokuapp.com` from some config variable?

Reply:
Done